src/controllers/trip.controller.js

```diff
@@ -124,24 +124,20 @@ class TripController {
   static async getTripRequestsByUser(req, res) {
     const userId = req.user.id;
     const { page } = req.query;
-    const limit = 10;
+    const limit = req.query.limit || 10;
     const offset = Paginate(page, limit);
     const requests = await tripService.findTripRequestsById(userId, limit, offset);
-    if (requests.count > offset) {
-      return response.successMessage(
-        res,
-        'My Trip Requests',
-        200,
-        requests
-      );
-    }
-    return response.errorMessage(
+    const manager = await db.user.findOne({ where: requests.managerId });
+    const requestTrips = await tripRequestService.getTripRequestsOfUser(requests, manager);
+    return response.successMessage(
```

Comment:
@VedaDom what if there are not trip requests for a particular user? That's why there was an if condition though the response status code returned was not correct (line 141)!

Reply:
I  wonted to return empty array when no records found.